lib/libzpool/uzfs_mtree.c

```diff
@@ -211,10 +211,9 @@ uzfs_txg_diff_tree_compare(const void *arg1, const void *arg2)
 	return (AVL_CMP(node1->offset, node2->offset));
 }
 
-
 int
 uzfs_get_txg_diff_tree(zvol_state_t *zv, uint64_t start_txg, uint64_t end_txg,
-    avl_tree_t **tree)
+    uzfs_zvol_traverse_t *func, void *arg)
```

Comment:
I think we should remove "tree" from function name for better clarity.

Reply:
ok